Document converting to plain Date (#245)
JuliaTime/TimeZones.jl: docs/src/conversions.md

end
```

> ## Converting to plain Dates types without timezone information.

Reviewer: ```suggestion
## Converting Date/Time without time zone information
```

> To convert from a `ZonedDateTime` into a vanillia `DateTime`, one can use the `DateTime` constructor.

Reviewer 2: ```suggestion
To convert from a `ZonedDateTime` into a vanilla `DateTime`, one can use the `DateTime` constructor.
```

> Passing either `Local` to directly drop the timezone,  or `UTC` to convert to UTC time first.

Reviewer: I'm hesitant to use the word "convert" as `UTC` is just returning a stored value where `Local` is where the conversion occurs:
```suggestion
Passing either `Local` to directly drop the time zone,  or `UTC` to extract the time as UTC instead.
```

> The canonical way to represent datetimes is generally in `UTC`, this is a requirement to correctly compute the [Unix Timestamp](https://en.wikipedia.org/wiki/Unix_time).

Reviewer 2: ```suggestion
The canonical way to represent datetimes is generally in `UTC`, as this is a requirement to correctly compute the [Unix Timestamp](https://en.wikipedia.org/wiki/Unix_time).
```

```jldoctest

> julia> x = ZonedDateTime(DateTime(2020, 11, 30, 22, 35), tz"America/Winnipeg")

Reviewer 2: ```suggestion
julia> x = ZonedDateTime(2020, 11, 30, 22, 35, tz"America/Winnipeg")
```

2020-11-30T22:35:00-06:00

julia> DateTime(x, Local)
2020-11-30T22:35:00

julia> DateTime(x, UTC)
2020-12-01T04:35:00

> ```

Reviewer: Can/should we use the same example as the docstring?

Similar can be done for `Date` and `Time`:

```jldoctest

> julia> x = ZonedDateTime(DateTime(2020, 11, 30, 22, 35), tz"America/Winnipeg")

Reviewer 2: ```suggestion
julia> x = ZonedDateTime(2020, 11, 30, 22, 35, tz"America/Winnipeg")
```
